src/modules/workspace/tabs/TasksTab.tsx

```
@@ -13,15 +13,16 @@
-import TaskCanvas from "../TaskCanvas";
+import { WorkspaceData, Task } from "../types/Types";
+import TaskCanvas from "../../task/TaskCanvas";
 import { X } from "lucide-react";
 
 interface TasksTabProps {
   workspaceData: WorkspaceData;
+  onTabChanges?: () => void;
 }
 
-const TasksTab: React.FC<TasksTabProps> = ({ workspaceData: workspaceData }) => {
+const TasksTab: React.FC<TasksTabProps> = ({ workspaceData: workspaceData, onTabChanges }) => {
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)